wolfssl/wolfcrypt/ecc.h

```diff
@@ -395,6 +401,12 @@ struct ecc_key {
 #if defined(PLUTON_CRYPTO_ECC) || defined(WOLF_CRYPTO_CB)
     int devId;
 #endif
+#ifdef WOLFSSL_SILABS_SE_ACCEL
+    sl_se_command_context_t  cmd_ctx;
+    sl_se_key_descriptor_t   key;
+    byte key_raw[3 * ECC_MAX_CRYPTO_HW_SIZE];
```

Comment:
Add comment. This is public X, public Y and private D offset by keysize.

Reply:
Let me know if the comment helps clarify.